code/style.css

```diff
@@ -1,4 +1,212 @@
+
+.container {
+    width: 320px;
+    margin: 25px auto;
 }
+
+/**** HEADER ****/
+.header-landing-page {
+    background: #13141b;
+    height: 100px;
+    display:flex;
+    align-items: center;
+    padding: 0 16px;
+    margin: -8px;
+    font-family: 'Staatliches', cursive;
+    justify-content: space-between;    
+}
+
+.header-logo {
+    width: 180px;
+}
+.header-logo-container {
+    overflow:hidden;
+    width: 100px;
+    height: 100px;
+    display: flex;
+    align-items: center;
+    justify-content: center;   
+}
+
+.foot-step-right-side {
+    margin:5px;
+    display: none;
+}
+
+.site-title {
+    color:white;
+    font-size: 58px;
+    display: flex;
+}
+.navigation-container {
+    display: none;
+    font-size: 24px;    
+    background-color: #13141b;
+    color:rgb(9, 85, 32);     
+}
+
+.navigation-item {
+    margin-right: 20px;    
+}
+
+.navigation-item:last-child {
+    margin-right: 0px;
+}
+
+.nav-link:any-link {
+    color:rgb(9, 85, 32); 
+}
+/**** BIG NEWS ****/
+
+.big-news {
+    display: flex;
+    justify-content: space-around;
+    text-align: center;  
+    border: 2px solid black;
+    align-items: center;
+    padding:30px 0;
+    background: whitesmoke;
+    flex-flow: column wrap;
+    margin: 70px 20px 20px 20px;
+    width:90%;
+}  
+
+.big-news:hover {
+    box-shadow: 18px 0 18px grey;
+}
+
+.big-news-text{
+
+    width:200px;
+}
+
+.big-news-img {
+    padding:20px 20px 30px 20px;
+    border:1px solid black;
+    box-shadow: 5px 10px 10px grey;
+    border-radius: 1px;
+    filter: grayscale(100%); 
+}
+
+.big-news-img:hover {
+    padding:20px 20px 30px 20px;
+    border:1px solid black;
+    box-shadow: 18px 10px 18px black;
+    filter: grayscale(0%);
+}
+.text-on-photo-big {
+    font-size: 50px;
+    margin-block-start: 10px;
+    margin-block-end: 1px;
+    font-family:'Indie Flower', cursive;    
+}
+
+/**** SMALL NEWS ****/
+
+.small-news {
+    display: flex; 
+    flex-flow:column wrap;
+        
+}
+  
+.small-news a {
+    display:flex;
+    flex-direction: column;
+    background: whitesmoke;  
+    margin: 10px 20px;    
+    text-align: center;  
+    width: 90%;  
+    border:2px solid black;    
+ }
+
+.small-news a:hover {
+
+    box-shadow: 18px 10px 18px grey;   
+  
+}
+
+    .small-news img {    
+    width:100%;    
+}
+
+
+  /***RESPONSIVE***/
+
+  @media (min-width: 768px) {
+
+    .container {
+        width: 768px;
+    }
+
+    .navigation-container {
+        display: flex; 
+                      
+    } 
+    .header-landing-page {
+        justify-content: space-between;
+    }
+        
+    .site-title {
+        display: none;
+    }
+    .big-news {
+        flex-direction: row;
+        width: 85%;   
+        align-items:stretch;     
+    }
+    .small-news {
+        flex-direction: row;
+        
+    }
+    .small-news a {
+        width: 40%;        
+        flex-direction: column;
+        display: flex;
```

Comment:
flex and flex direction is already set on the "main" rules for these elements on line 115. No need to specify it here again. It's inherited. just the things that are changing should be in here